src/main/webapp/add-account.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
```

Comment:
I think there are places in this file that we increase code reuse from other pages that is easier to maintain. I don't know the best way of doing it, but I feel like it is worth considering

Reply:
I struggled with that for a while. I agree with you, but I wanted to get something functional for the MVP.